music21/musicxml/xmlToM21.py

```diff
@@ -4363,9 +4363,11 @@ def findM21VoiceFromXmlVoice(self, mxVoice=None):
         if not textStripValid(mxVoice):
             useVoice = self.lastVoice
             if useVoice is None:
-                environLocal.warn('Cannot put in an element with a missing voice tag when '
-                                  + 'no previous voice tag was given.  Assuming voice 1... '
-                                  )
+                # # Commenting this out -- this is very common with SmartScore OMR software.
+                environLocal.printDebug(
+                    'Cannot put in an element with a missing voice tag when '
+                    + 'no previous voice tag was given.  Assuming voice 1... '
+                )
```

Comment:
Is this still a problem after #1030 ? If there's another failure case, we could look into why.

Reply:
Ah!  I thought that this might be obsolete -- will take it out.

This was a left-over uncommitted set of changes from a long time ago that I wanted to get in so I had a clean working directory for #1054